javascript/javascript1/week1/goodbye-oldboy.js

```diff
@@ -0,0 +1,13 @@
+
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+
+    if (shouldShowResultInDogYears === true) 
```

Comment:
the `if` statement is anyways expecting a result that's true. As such, you don't need to explicitly write `=== true` and can just do:
```suggestion
    if (shouldShowResultInDogYears) 
```

Reply:
Got the point. Done it.